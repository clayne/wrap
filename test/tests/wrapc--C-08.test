wrapc | /dev/null | | test_08.c | 0
