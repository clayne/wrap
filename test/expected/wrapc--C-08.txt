    // When wrapped, there should be no additional line added.
    //
    //
    printf( "hello, world\n" );
